conf/deployment/aws/ipi_3az_rhcos_3m_6w.yaml

```diff
@@ -0,0 +1,4 @@
+---
+ENV_DATA:
+  worker_replicas: 6
+  worker_instance_type: 'm5.4xlarge'
```

Comment:
If this is going to be production config I would like to keep all the details like we have here: https://github.com/red-hat-storage/ocs-ci/blob/6fab465c910f88a999c81a8eb271c73ba26b11d5/conf/deployment/aws/ipi_3az_rhcos_3m_3w.yaml just increase number of nodes + nodes which suppose to be labeled to 6 as well.